docs/CONTRIBUTING.md

```diff
@@ -72,6 +72,11 @@ the following tools by hand:
 - [`pg_dump`](https://stackoverflow.com/a/49689589)
   - on macOS, run `brew install libpq zstd`
   - on Linux, install [`zstd`](https://github.com/horta/zstd.install)
+- PostgreSQL 13 (optional if Docker is available)
+  - *Note*: If you are using Docker, you can skip this step
+  - on macOS, run `brew install postgresql@13` and `brew services start postgresql@13`
+  - To enable schema generation with non-containerized PostgreSQL, set the following environment variable:
+    - `export DB_DUMP_CONNECTION_URL="postgres://postgres@localhost:5432/postgres?password=postgres&sslmode=disable"`
```

Comment:
Maybe add a warning note regarding `pg_dump` version compatibility:

```
DB_DUMP_CONNECTION_URL='postgresql://user:pass@ds918.home:5433/coder_tmp?sslmode=disable' make gen
pg_dump: error: server version: 16.2 (Debian 16.2-1.pgdg120+2); pg_dump version: 13.18
pg_dump: error: aborting because of server version mismatch
panic: dump schema failed: exit status 1

goroutine 1 [running]:
main.main()
        /Users/cian/src/coder/coder/coderd/database/gen/dump/main.go:70 +0x354
exit status 2
make: *** [Makefile:618: coderd/database/dump.sql] Error 1
```

Reply:
I think this is an unlikely scenario, you can use `pg_dump@16` with `postgresql@13` but not vice-versa as you saw here. Since we're advising to use an older postgresql server here, you'd really have to make an effort to use an even older `pg_dump` 😄